### ISSUE TITLE
Clicking on Add filters refresh the page and render filter disappear 

### ISSUE THREAD

**Issue by panday1995** [microsoft/vscode-data-wrangler#251]
Type: <b>Bug</b>

# Behaviour

## Expected vs. Actual

Expected:

Click Add filter to add filters to a column

Actual:

After clicking add filter, the page is refreshed and popup window of add condition is not showing up

## Steps to reproduce:

1. XXX

# Diagnostic data

-   Jupyter extension version: 2024.6.0
-   Python extension version: 2024.12.1
-   .NET Install Tool for Extension Authors extension version: Not installed
-   Python package dependencies:

```json
{
  "installed": {
    "pandas": "2.0.3",
    "pyarrow": "16.1.0"
  },
  "required": {
    "pandas": "1.2.0"
  },
  "unsatisfied": []
}
```

-   Entrypoint: JupyterNotebookToolbar
-   Active mode: dataViewer

Extension version: 1.6.0
VS Code version: Code 1.91.1 (f1e16e1e6214d7c44d078b1f0607b2388f29d729, 2024-07-09T22:06:49.809Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz (8 x 2419)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.73GB (6.89GB free)|
|Process Argv|. --crash-reporter-id 4b8e2f00-6f79-490f-9035-e0a5846c2b03|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscorecescf:30445987
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
vscaat:30438848
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonnoceb:30805159
asynctok:30898717
pythonregdiag2:30936856
pythonmypyd1:30879173
2e7ec940:31000449
pythontbext0:30879054
accentitlementsc:30995553
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
0ee40948:31013168
pythoncenvpt:31062603
a69g1124:31058053
dvdeprecation:31068756
dwnewjupyter:31046869
2f103344:31071589
impr_priority:31102340
refactort:31108082
ccpltc:31103427
pythonrstrctxt:31103193
wkspc-ranged-t:31107835

```

</details>

**Comment by pwang347** [comment]
Hi @panday1995, thanks for reporting! We'll investigate the issue and update this thread. In the meantime, it would help if you could provide more information about how to reproduce this, e.g. if you have any specific code used to launch the viewer that you can share. Is this issue also happening every time the "Add filter" context menu button is pressed or only in some scenarios?

**Comment by pwang347** [comment]
Hi @panday1995! From what I am seeing this issue should be already fixed in the latest version 1.6.0, unless I am misunderstanding your original issue report. I will close the issue for now, but please feel free to reopen if you could provide more information on how to reproduce it (it would also be great to see a quick GIF of what you are experiencing).

Thanks!
